client/src/common_text/common_lang.yaml

```diff
@@ -1122,6 +1122,13 @@ infographic_for:
   en: Infographic for {{subject.name}}
   fr: Infographie pour {{subject.name}}
 
+department_of:
+  transform: [handlebars]
+  en: |
+    {{subject.name}} ({{subject.dept.applied_title}})
+  fr: |
+    {{subject.name}} ({{subject.dept.applied_title}})
+
```

Comment:
Tried to truncate unnecessary words like "Department of" and "Infographic for" for programs, but some are still too long and get cut off on the website.